packages/duckdb/index.cjs

```diff
@@ -42,7 +42,12 @@ const mapResultsToEvidenceColumnTypes = function (rows) {
 
 const runQuery = async (queryString, database) => {
 	const filename = database ? database.filename : getEnv(envMap, 'filename');
-	const filepath = filename !== ':memory:' ? '../../' + filename : filename;
+	var filepath;
+	if (filename.includes('.db') || filename.includes('.duckdb')) {
```

Comment:
Not sure if this is exactly the right logic to be most future proof here.

This will catch all possible duckdb files that people add via the UI. 

I guess it's possible someone could have a .db in their token name, or that it could be a duckdb file stored in an S3 bucket or similar

Reply:
this should not be breaking at least